mnist/main.py

```diff
@@ -57,7 +57,9 @@ def main():
     np.random.seed(seed)
 
     # Load the data
-    train_images, train_labels, test_images, test_labels = map(mx.array, mnist.mnist())
+    train_images, train_labels, test_images, test_labels = map(
+        mx.array, mnist.fashion_mnist() if args.fashion_mnist else mnist.mnist()
```

Comment:
If you make the argument change below, then you can change this to

```suggestion
        mx.array, getattr(mnist, args.dataset),
```